City-of-Helsinki/drupal-hdbt
# Added transliteration

## What was done
Copied transliteration logic from toc.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-X_scandic_character_id_fix`
* Run `make drush-cr`

## How to test
- Go to page with toc
 - Toc must have titles with ä or ö characters
- click toc link with ä or ö characters
- anchor link works as expected
